python/ray/tests/test_gcs_ha_e2e.py

```diff
@@ -233,9 +233,9 @@ def check_alive(n):
     # When GCS restarts, a new raylet is added
     # and the old dead raylet is going to take a while to be marked dead.
     # So there should be 3 alive nodes
-    wait_for_condition(check_alive, timeout=10, n=3)
+    wait_for_condition(check_alive, timeout=2, n=3)
```

Comment:
hmm, why does a longer time out cause issue?

Reply:
@jjyao it's because it now only takes 5 seconds to detect the GCS is changed and raylet kills itself.

But actually I am curious how this works with 30 seconds delay because the original membership checking interval was 60 seconds. cc @iycheng?

Reply:
Hi Sang, this is to test that GCS will mark raylet dead, not the other way.